lightning/exports.sh

```diff
@@ -21,6 +21,9 @@ BIN_ARGS+=( "--tlsautorefresh" )
 # lnd.conf for existing users via a migration.
 BIN_ARGS+=( "--accept-amp" )
 
+# Lightning Terminal (litd) now requires this flag to be set or it will not startup
+BIN_ARGS+=( "--rpcmiddleware.enabled" )
```

Comment:
Potentially a typo here? My `lnd` binary says the flag is: `--rpcmiddleware.enable`

Reply:
Ah, typo. Thanks.

Fixed in https://github.com/getumbrel/umbrel-apps/pull/249/commits/7002ebc92a59ec128518671cc391dd1e85e8105b